exonum/src/events/noise/wrappers/sodium_wrapper/wrapper.rs

```diff
@@ -143,13 +144,14 @@ impl NoiseWrapper {
         Ok(None)
     }
 
-    fn read(&mut self, input: &[u8], len: usize) -> Result<Vec<u8>, NoiseError> {
+    pub fn read(&mut self, input: &[u8], len: usize) -> Result<Vec<u8>, NoiseError> {
```

Comment:
If we mark method as public then it's a good practice to add some docs to describe what does this method, isn't it ?

Reply:
It's a typo, I forgot to revert this method to private.